crates/wasmcloud-provider/src/lib.rs

```diff
@@ -356,6 +388,7 @@ struct VolumeBinding {
 struct Capability {
     name: &'static str,
     binding: Option<String>,
+    capability_provider_id: &'static str,
```

Comment:
nit: should this be shortened to `provider_id`?

Reply:
I thought that would lead to confusion with the fact that this is _also_ a provider, but of the Krustlet variety

Reply:
ooh yeah good call. I guess a little diasmbiguation is good here.